### PR TITLE
56.fre pp configure updates

### PR DESCRIPTION
**Pull request by singhd789**

### CODE REVIEW

File: fre/frepp/configureScriptYAML.py [hunk 1]
```diff
@@ -23,66 +24,46 @@ def validateYaml(file):
...
   p = platform
-  t = target 
+  t = target
+  yml = yamlfile
 
-  with open(y,'r') as f:
+  with open(yml,'r') as f:
     y=yaml.safe_load(f)
-    yml = validateYaml(y)
+    valyml = validateYaml(y)
```
Comment (ceblanton) [inline fre/frepp/configureScriptYAML.py:35]: `y` is no longer set.. should this be `validateYaml(yml)` instead?

**Review by ceblanton — CHANGES_REQUESTED** [review]
Looks good. I think you renamed `y` to `yml` in one section that needs one more change, though.

File: fre/frepp/configureScriptYAML.py (hunk 1 above)
Comment (singhd789) [reply fre/frepp/configureScriptYAML.py:35]: The yamlfile is being saved as `yml`, which is being opened as `f` and then `safe_loaded` as `y`. So the loaded yaml is still `y`.

Comment (singhd789) [reply fre/frepp/configureScriptYAML.py:35]: Before I had overwritten variables accidentally, using `y` multiple times I think

Comment (ceblanton) [reply fre/frepp/configureScriptYAML.py:35]: Gotcha!

**Review by ceblanton — APPROVED** [review]

**Pull request merged**